### PR TITLE
Switched coordinates from a map to THREE.Vector3

### PR DESCRIPTION
**Pull request by jamesmuerle**

### CODE REVIEW

File: game/static/game/js/game.js [hunk 1]
```diff
@@ -265,6 +265,7 @@ Game.prototype.handleClickMessage = function (data) {
     var timestamp = data['timestamp'];
     var player_id = data['player_id'];
     var clickpos = data['clickpos'];
+    clickpos = new THREE.Vector3(clickpos.x, clickpos.y, clickpos.z);
```
Comment (hellochar) [inline game/static/game/js/game.js:268]: clickpos has a `z`? I thought it was just screen coordinates

File: jasmine_testing/static/jasmine_testing/specs/GameSpec.js [hunk 2]
```diff
@@ -213,7 +213,7 @@ describe("Game", function() {
         beforeEach(function() {
             game.player_id = 120;
             game.game_id = 9999;
-            game.handleClick(1, {x: 10, y:100});
+            game.handleClick(1, new THREE.Vector3(10, 100));
```
Comment (hellochar) [inline jasmine_testing/static/jasmine_testing/specs/GameSpec.js:216]: What happens to the `z` coordinate if no value is passed? What's the advantage over just using a Vector2?

**Comment by hellochar** [comment]
Also we should talk with everyone before making the design decision to have full 3D in the game model, because I still just want 2D with a height

File: game/static/game/js/game.js (hunk 1 above)
Comment (jamesmuerle) [inline game/static/game/js/game.js:268]: In the future the clickpos will in fact be game coordinates, not screen coordinates, so it makes sense to have a Vector3 here to hold the place for now.

File: jasmine_testing/static/jasmine_testing/specs/GameSpec.js (hunk 2 above)
Comment (jamesmuerle) [inline jasmine_testing/static/jasmine_testing/specs/GameSpec.js:216]: Default value for all Vector3 arguments is 0. The advantage of this over a Vector2 is that a z coordinate still exists though it is not explicitly assigned in the constructor.

**Comment by hellochar** [comment]
lg2m

**Pull request merged**